### PR TITLE
added anonymous auth

### PR DESCRIPTION
**Pull request by jeweller1980stepanets**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Due to a fact the cordova-neura-plugin is based on NeuraSDK.framework, and the f
 	'cordova plugin add /path/to/cordova-neura-plugin/'
 9. Build cordova project (--verbose can be omitted here): 
  	'cordova build ios --verbose'
+	
+	# TODO: delete it. I added it functional to plugin.xml file
```
Comment (youvalv) [inline README.md]: which functionality in the plugin.xml?

File: src/android/neura.java [hunk 2]
```diff
@@ -156,163 +175,234 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
...
-    }
 
+
+
+
+
+
+    }
+    
+    private void registerPushServerApiKey(JSONArray args, CallbackContext callbackContext) {
```
Comment (youvalv) [inline src/android/neura.java]: You can delete this deprecated method

File: src/android/neura.java [hunk 3]
```diff
@@ -323,205 +413,204 @@ public void onSuccess(List<AppSubscription> list) {
...
         }
     }
-
+    
     private void enableAutomaticallySyncLogs(JSONArray args, CallbackContext callbackContext) {
         try {
             boolean enableAutomaticallySyncLogs = args.getBoolean(0);
-            mNeuraApiClient.enableAutomaticallySyncLogs(enableAutomaticallySyncLogs);
-
+            //mNeuraApiClient.enableAutomaticallySyncLogs(enableAutomaticallySyncLogs);
```
Comment (youvalv) [inline src/android/neura.java]: Why is this commented out?

File: src/android/neura.java (hunk 2 above)
Comment (jeweller1980stepanets) [reply src/android/neura.java]: ok. i will do it in new pull request

File: src/android/neura.java (hunk 3 above)
Comment (jeweller1980stepanets) [reply src/android/neura.java]: this method is deprecated. I can delete it.

**Pull request merged**